src/routes.js

```diff
@@ -4,15 +4,28 @@ const index = require('./routes/index');
 const artists = require('./routes/artists');
 const albums = require('./routes/albums');
 const albumsForArtists = require('./routes/albumsForArtists');
+const users = require('./routes/users');
+const session = require('./routes/session');
 
 const router = new KoaRouter();
 
+router.use(async (ctx, next) => {
+  if (ctx.session.currentUserId) {
+    ctx.state.currentUser = await ctx.orm.user.findByPk(ctx.session.currentUserId);
+  }
+  return next();
+});
+
 router.use(async (ctx, next) => {
   ctx.state.paths = {
     index: ctx.router.url('index'),
+    users: ctx.router.url('users.list'),
     artists: ctx.router.url('artists.list'),
     albums: ctx.router.url('albums.list'),
     about: ctx.router.url('index.about'),
+    newSession: ctx.router.url('session.new'),
+    destroySession: ctx.router.url('session.destroy'),
+    profilePath: ctx.session.currentUserId && ctx.router.url('users.show', { id: ctx.session.currentUserId }),
```

Comment:
Aquí podría omitirse el sufijo `Path`, para ser consistente con el resto (ya estamos "dentro" del objeto `ctx.state.paths`):
```suggestion
    profile: ctx.session.currentUserId && ctx.router.url('users.show', { id: ctx.session.currentUserId }),
```